dev/reference/mysql-compatibility.md

```diff
@@ -149,3 +149,23 @@ TiDB supports **all of the SQL modes** from MySQL 5.7 with minor exceptions:
         - For MySQL 5.7: `OFF`
         - For MySQL 8.0: `ON`
     
+### Date and Time
+
+#### Named timezone
+
+TiDB supports named timezones such as `America/Los_Angeles` without having to load the [time zone information tables](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html#time-zone-installation) as in MySQL.
+
+Because they are built-in, named time zones in TiDB may behave slightly differently to MySQL, and cannot be modified. For example, in TiDB the names are case-sensitive [#8087](https://github.com/pingcap/tidb/issues/8087).
```

Comment:
Let's use "might" instead of "may" in the documentation, because "may" has multiple meanings.

```suggestion
Because they are built-in, named time zones in TiDB might behave slightly differently to MySQL, and cannot be modified. For example, in TiDB the names are case-sensitive [#8087](https://github.com/pingcap/tidb/issues/8087).
```